### PR TITLE
[develop] Integrate ov-telemetry

### PR DESCRIPTION
**Pull request by yunchu**
openvino telemetry integration

once it enabled on the consent file at `$HOME/intel/openvino_telemetry`, all cli command executions will be sent to ov-telemetry backend.

### CODE REVIEW

**Comment by yunchu** [comment]
run ote-test

File: README.md [hunk 1]
```diff
@@ -85,6 +85,29 @@ Please read the [Contribution guide](CONTRIBUTING.md) before starting work on a
 
 ---
 
+## Telemetry data collection note
+
+The [OpenVINO™ telemetry library](https://github.com/openvinotoolkit/telemetry/)
+is used to collect basic information about usage of this toolkit.
+
+The table below shows which event and data would be sent through telemetry data collection tool.
```
Comment (sungmanc) [inline README.md:93]: The table below --> The below table, looks good to me

File: ote_cli/ote_cli/utils/telemetry.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+from ote_cli import get_version
+
+try:
+    import openvino_telemetry as tm
+except ImportError:
+    from ote_cli.utils import telemetry_stub as tm
+
+
+__TM_CATEGORY_OTX = "otx"
+__TM_MEASUREMENT_ID = "UA-17808594-29"
```
Comment (sungmanc) [inline ote_cli/ote_cli/utils/telemetry.py:23]: Is it okay to use ID on code, explicitly?

Comment (yunchu) [reply ote_cli/ote_cli/utils/telemetry.py:23]: i'm not sure but other tool like Datumaro also include the ID as a code.

**Review by sungmanc — APPROVED** [review]

**Review by harimkang — APPROVED** [review]

**Comment by yunchu** [comment]
run ote-test

**Pull request merged**